Library/macOSRichTextEditor/Source/RichTextEditor.m

```diff
@@ -1232,8 +1291,8 @@ - (void)keyDown:(NSEvent*)event {
         }
         
         unichar keyChar = 0;
-        bool shiftKeyDown = event.modifierFlags & NSShiftKeyMask;
-        bool commandKeyDown = event.modifierFlags & NSCommandKeyMask;
+        bool shiftKeyDown = event.modifierFlags & NSEventModifierFlagShift;
+        bool commandKeyDown = event.modifierFlags & NSEventModifierFlagCommand;
```

Comment:
This change has nothing to do with this PR. In open source, your PR needs to stay on topic. If you think this is an important item, good practice is to either:

* Open a separate issue
* Open a separate pull request with **just** this change

Otherwise, you run the risk of breaking someone's code with a small change that "sneaked" in.

Reply:
Yes that was a total mistake. I did not mean to commit it as part of this pull request